doc/customization_basic1.rst

```diff
@@ -0,0 +1,109 @@
+.. SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _customization_basic1:
+
+#######################
+2. Basic customization
+#######################
+
+A good starting point to customize your model are settings of the default configuration file `config.default`. You may want to do a reserve copy of your current configuration file and then overwrite it by a default configuration:
```

Comment:
Could make sense to add an extension to the file name: `config.default.yaml` (not crucial, just for consistency)